data/shreyas0924.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "Shreyas P",
+  "bio": "Full-Stack Developer",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Shreyas0924",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/shreyas0924",
+      "icon": "FaGithub"
+    },
```

Please remove this trailing comma here. 

```suggestion
    }
```